src/main/java/com/spotify/reaper/cassandra/JmxProxy.java

```diff
@@ -58,10 +58,13 @@
 import com.google.common.collect.Lists;
 
 import com.datastax.driver.core.policies.EC2MultiRegionAddressTranslator;
+import com.spotify.reaper.ReaperApplication;
 import com.spotify.reaper.ReaperException;
 import com.spotify.reaper.core.Cluster;
 import com.spotify.reaper.service.RingRange;
 
+import jersey.repackaged.com.google.common.collect.Maps;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'jersey.repackaged.com.google.common.collect.Maps'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUselessImportCheck)